api/src/main/java/org/cardanofoundation/rosetta/api/network/service/NetworkServiceImpl.java

```diff
@@ -163,47 +146,24 @@ public Network getSupportedNetwork() {
     };
   }
 
-  private NetworkStatus networkStatus() throws IOException {
+  private NetworkStatus networkStatus() {
     log.info("[networkStatus] Looking for latest block");
-    Block latestBlock = ledgerBlockService.findLatestBlock();
-    log.debug("[networkStatus] Latest block found {}", latestBlock);
+    BlockIdentifierExtended latestBlock = ledgerBlockService.findLatestBlockIdentifier();
+    log.debug("[networkStatus] Latest block found " + latestBlock);
+
     log.debug("[networkStatus] Looking for genesis block");
-    GenesisBlock genesisBlock = ledgerBlockService.findGenesisBlock();
-    log.debug("[networkStatus] Genesis block found {}", genesisBlock);
+    BlockIdentifierExtended genesisBlock = ledgerBlockService.findGenesisBlockIdentifier();
+    log.debug("[networkStatus] Genesis block found " + genesisBlock);
```

Comment:
Did you change the parametrized string intentionally?

Reply:
No, fixed